src/printapp/printstatus.py

```diff
@@ -176,9 +181,38 @@ def fetch_data(self):
                 raise ScrapingError(err)
             except ValueError as err:
                 raise ScrapingError(err)
-            print_jobs.append(PrintJob(name, pages, copies, price, printer_name, date))
+            job_id = re.search(r"c_OnSelectJob\('(.*)'\)", onclick_attribute).group(1)
+            if job_id is None:
+                raise ScrapingError("Print job id not found for document: {}".format(name))
```

Comment:
Let's put these three lines up by line 173, since `onclick_attribute` isn't really a thing, so it's unclear what it's for since we're not using it right away.